pkg/controller/config_watcher.go

```diff
@@ -47,19 +47,21 @@ func (w *ConfigWatcher) Do(ctx context.Context, cancelFunc context.CancelFunc) (
 	ctx, cancelFn := context.WithCancel(ctx)
 	defer cancelFn()
 
+	log := w.log.WithField("configs", w.configPaths)
```

Comment:
Ignore if you don't like it, but personally I would keep the same name when logging this as the property one 
```suggestion
	log := w.log.WithField("configPaths", w.configPaths)
```